bitrise.yml

```diff
@@ -685,7 +676,7 @@ workflows:
     description: This Workflow is to build the app using latest xcode available in Bitrise
     meta:
       bitrise.io:
-        stack: osx-xcode-15.0.x
+        stack: osx-xcode-osx-xcode-14.3.x
```

Comment:
👎🏼 We should not allow this change. Our code base has been updated to XCode 15.x.